test/Libraries/RevitTestServices/RevitSystemTestBase.cs

```diff
@@ -221,12 +222,18 @@ protected override void StartDynamo(TestSessionConfiguration testConfig)
 
                 Model = DynamoRevit.RevitDynamoModel;
 
-                this.ViewModel = DynamoViewModel.Start(
+                this.ViewModel = DynamoRevitViewModel.Start(
```

Comment:
That's a good question. I am not sure why this was done originally, but it seemed that we should be testing the way that a user would be running, using `DynamoRevitViewModel`. It is possible that when we run this through testing, the reason for this change will become obvious.